www/docs/src/content/docs/start-here/getting-started.mdx

```diff
@@ -89,15 +89,44 @@ export default defineConfig({
 });
 ```
 
+## Configure your `package.json`
+
+Setup your package.json scripts to include the `--remote` flag for build, and optionally for dev. (You can also run the dev command as shown in the "Running your StudioCMS Project" section)
+
+```json title="package.json"
+{
+    "name": "my-studiocms-project",
+    "scripts": {
+        "dev": "astro dev --remote",
+        "build": "astro check & astro build --remote",
+        "astro": "astro"
+    }
+}
+```
+
 :::note
 Astro StudioCMS requires SSR mode to work correctly. Make sure to set `output: 'server'` and an adapter that supports SSR in your `astro.config.mjs` file.
 :::
```

Comment:
This should be under the previous header as it relates to the Astro config, not the `package.json` file

Reply:
whoops... working from my windows install at the moment so I'm on the web browser and didn't notice that got put in the wrong place... 😅 fixed in [951dce1](https://github.com/astrolicious/studiocms/pull/267/commits/951dce1f83a75eb330c02552d386388ea47f67f7)